web/backend/src/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: 'dvoting'
+      POSTGRES_PASSWORD: 'dvoting'
+    volumes:
+      - ./data:/var/lib/postgresql/data
+      - ./mysql.sql:/docker-entrypoint-initdb.d/init.sql

```

Comment:
`mysql.sql` is not a good name. You are not using a mySQL database. `migration.sql` would be a better name.